code/src/components/HappyListItem.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { formatDistanceToNow } from 'date-fns'
+import Emoji from './Emoji'
+
+const HappyListItem = ({ thoughtId, message, likes, creationDate, onHeartClick }) => {
+
+    const creationDateObject = new Date(creationDate)
+    const formattedCreationDate = formatDistanceToNow(creationDateObject, { addSuffix: true })
+
+    // Two different heart backgrounds depending on if there are likes or not
```

Comment:
Thanks for commenting this, so we know why there is a different design for the hearts.